### ISSUE TITLE
Hardware volume buttons should work from all activities

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#46]
```
The hardware volume buttons only work on the "now playing" activity.

They should work on all activities.
```

Original issue reported on code.google.com by `nik@ngo.org.uk` on 2011-09-24 09:25:11

**Comment by kaaholst** [comment]
```
Fixed in 0.6.1.
```

Original issue reported on code.google.com by `nik@ngo.org.uk` on 2012-01-22 09:13:33

**Comment by kaaholst** [comment]
```
Fixed in 0.6.1.
```

Original issue reported on code.google.com by `nik@ngo.org.uk` on 2012-01-22 09:13:33
